src/Features/VisualBasic/Portable/SimplifyThisOrMe/VisualBasicSimplifyThisOrMeCodeFixProvider.vb

```diff
@@ -23,24 +23,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.SimplifyThisOrMe
         End Function
 
         Protected Overrides Function Rewrite(semanticModel As SemanticModel, root As SyntaxNode, memberAccessNodes As ISet(Of MemberAccessExpressionSyntax)) As SyntaxNode
```

Comment:
@CyrusNajmabadi Can we get rid of semanticModel being passed down at any higher level?

Reply:
Will look into doing that!